docs/contributing/index.md

```diff
@@ -1,310 +1,153 @@
 
-    ```shell
-    # Optionally clean the builds to avoid cache issues
-    make clean
-    make html
-    make linkcheck
-    ```
 
-    ```{note}
-    Currently there are some errors on the `html` build, mostly due to empty `meta` HTML tags.
-    You are welcome to fix as many errors as you like.
-    You are only responsible to fix errors that you create.
-    ```
+(contributing-change-log-label)=
 
-    ```{note}
-    Eventually the `vale` build will be required, but not at this time.
-    We welcome improvements to the dictionary.
-    ```
+## Change log entry
 
-    ```{seealso}
-    {ref}`setup-build-available-documentation-builds-label`.
-    ```
+Plone projects require that you include a change log entry or news item with your contribution.
+This is enforced by continuous integration through GitHub Actions.
 
-1.  After the builds pass, commit changes to your branch, and push it to the remote repository on GitHub.
-    The remote repository may be either a branch in your fork of the project or a branch in the `plone/documentation` upstream repository.
+Plone uses [`towncrier`](https://github.com/collective/zestreleaser.towncrier) to manage change log entries and to automatically generate history or change log files from the entries.
+The log file is usually named `CHANGES.rst`, `CHANGES.md`, or `CHANGELOG.md`, and is located at the root of the project.
+When a project is released with a new version, the release manager runs `towncrier` as part of the release process.
+Because the log file is automatically generated, you should not edit it directly, except to make corrections, such as broken links.
 
-    ```shell
-    git commit -m "My descriptive commit message"
-    git push
-    ```
+To create a change log entry or news item, create a file that is placed in the root of the repository directory at `/news`.
+Its format must be `###.type`, where `###` is the referenced GitHub issue or pull request number, `.` is the literal extension delimiter, and `type` is one of the following strings.
 
-1.  Visit the GitHub `documentation` repository, and [create a pull request](https://github.com/plone/documentation/compare) against the branch `6-dev`.
-1.  Members who subscribe to the repository will receive a notification and review your request.
-1.  Request a review from other team members.
+-   `breaking` for breaking changes
+-   `bugfix` for bug fixes
+-   `documentation` for documentation
+-   `feature` for new features
+-   `internal` for internal changes
 
+A project configures the types it allows in a file `towncrier.toml` located at the root of its repository.
 
-(contributing-editing-external-package-documentation-label)=
+The content of this file must include the following.
 
-### Editing external package documentation
+-   A brief message that summarizes the changes in your contribution.
+-   An attribution to yourself, either in the format of `@github_username` or `[github_username]`.
```

Comment:
Not for here, but as a note: This should be unified.

Reply:
I would say, yes for here and now, let's standardize. It's frustrating to have deviations across the organization.

Reply:
Arbitrary executive decision by me: `@github_username`.